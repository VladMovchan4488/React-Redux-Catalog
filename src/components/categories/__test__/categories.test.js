import React from "react";
import Categories from "../categories";
import Adapter from "enzyme-adapter-react-16";
import { shallow, configure } from "enzyme";
import toJson from "enzyme-to-json";

configure({ adapter: new Adapter() });

const props = {
  category: [],
  products: [],
  currentCategory: "",
  getCategoryArr() {},
  setCategoryValue() {},
  history: {
    location
  }
};

describe("test Search", () => {
  const wrapper = shallow(<Categories.WrappedComponent {...props} />);

  it("TEST spapshot to component", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
