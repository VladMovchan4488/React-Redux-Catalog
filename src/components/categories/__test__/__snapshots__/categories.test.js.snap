// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Search TEST spapshot to component 1`] = `
<div>
  <styled.div>
    <NavLink
      onClick={[Function]}
      to="/?"
    >
      <h5>
        All Category
      </h5>
    </NavLink>
  </styled.div>
</div>
`;
